feature-libs/asm/customer-360/components/customer-360/customer-360.component.html

```diff
@@ -162,7 +162,24 @@ <h3 id="asm-customer-360-title" class="modal-title">
 
     <!-- Modal Body -->
     <div class="cx-dialog-body">
-      <div class="cx-tab-headers" role="tablist">
+      <div class="cx-tab-headers" role="tablist" *cxFeatureLevel="'!6.5'">
+        <ng-container *ngFor="let tabHeader of tabs; let i = index">
+          <button
+            #tabHeaderItem
+            class="cx-tab-header"
+            role="tab"
+            [attr.aria-selected]="activeTab === i"
+            attr.aria-controls="asm-customer-360-tab-panel"
+            (click)="selectTab(i)"
+            [class.active]="activeTab === i"
+            (keydown)="switchTab($event, i)"
+            *ngIf="tabHeader.i18nNameKey !== 'customer360.promotionsTab'"
```

Comment:
if we has more sections, we'll add  && not xxx, not yyy and not zzz?

Why not *ngIf="tabHeader.i18nNameKey === 'customer360.xxx'"

Reply:
Cause there is already a `*ngFor` and so many tabs need to be displayed, at the mean time only promotion tab need to be hided before 6.5.